src/Data/Uniques/sword.lua

```diff
@@ -677,27 +644,25 @@ Variant: Pre 2.6.0
 Variant: Pre 3.11.0
 Variant: Pre 3.17.0
 Variant: Current
-Requires Level 59, 82 Str, 119 Dex
```

Comment:
This Dancing Duo could be listed as no longer obtainable while we're in here

Reply:
It already is listed as no longer obtainable on line 663.

Reply:
Sorry, could've sworn I was in the Dancing Duo when I wrote this